scripts/sql/apolloportaldb.sql

```diff
@@ -381,3 +381,51 @@ CREATE TABLE SPRING_SESSION_ATTRIBUTES (
 /*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
 /*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+
+
+DROP TABLE IF EXISTS `AuditLog`;
+
+CREATE TABLE `AuditLog` (
+  `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '主键',
+  `TraceId` varchar(32) NOT NULL DEFAULT '' COMMENT '链路全局唯一ID',
+  `SpanId` varchar(32) NOT NULL DEFAULT '' COMMENT '跨度ID',
+  `ParentSpanId` varchar(32) DEFAULT NULL COMMENT '父跨度ID',
+  `FollowsFromSpanId` varchar(32) DEFAULT NULL COMMENT '上一个兄弟跨度ID',
+  `Operator` varchar(64) NOT NULL DEFAULT 'anonymous' COMMENT '操作人',
+  `OpType` varchar(50) NOT NULL DEFAULT 'default' COMMENT '操作类型',
+  `OpName` varchar(150) NOT NULL DEFAULT 'default' COMMENT '操作名称',
+  `Description` varchar(200) DEFAULT NULL COMMENT '备注',
+  `IsDeleted` bit(1) NOT NULL DEFAULT b'0' COMMENT '1: deleted, 0: normal',
+  `DeletedAt` BIGINT(20) NOT NULL DEFAULT '0' COMMENT 'Delete timestamp based on milliseconds',
+  `DataChange_CreatedBy` varchar(64) NOT NULL DEFAULT 'default' COMMENT '创建人邮箱前缀',
+  `DataChange_CreatedTime` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT '创建时间',
+  `DataChange_LastModifiedBy` varchar(64) DEFAULT '' COMMENT '最后修改人邮箱前缀',
+  `DataChange_LastTime` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT '最后修改时间',
+  PRIMARY KEY (`Id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COMMENT='审计日志表';
+
+CREATE INDEX idx_trace_id_time ON AuditLog (TraceId, DataChange_CreatedTime);
```

Comment:
CREATE INDEX in CREATE TABLE

for example
```
UNIQUE KEY `UK_Key_TraceId` (`TraceId`,`DeletedAt`),
```

by the way, traceId doesn't need create time, 

user maybe search by OpName, time, or operator, so the index maybe
```
KEY `IX_OpName` (`OpName`),
KEY `IX_DataChange_CreatedTime` (`DataChange_CreatedTime`),
KEY `IX_Operator` (`Operator`),
```

Reply:
When design the index for queryDataInfluenceByEntityIdAndEntityNameAndFieldName, i don't know whether union index like 
```java
KEY `IX_Entity_Field` (`InfluenceEntityName`, `InfluenceEntityId`, `FieldName`)
```
or uses three individual indexs

Reply:
Depends on how UI search

Reply:
queryDataInfluenceByEntityIdAndEntityNameAndFieldName

entityId -> entityName -> fieldName

I think only need a single index with entityId